src/R4Mvc/ModelUnbinders/SimpleModelUnbinder.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Routing;
+using R4Mvc.ModelUnbinders;
+
+namespace R4Mvc.ModelUnbinders
+{
+    public class SimplePropertyModelUnbinder : IModelUnbinder
```

Comment:
I'm not sure whether it should be part of R4Mvc (they didn't add it to T4MVC, for example). While good in some scenarios, I don't think it's a great practice to automatically unbind model classes like that. Plus, T4MVC doesn't have it, and we're trying to achieve as much feature parity as possible. It can be moved to the host project though, for testing.